spec/features/applications/new_spec.rb

```diff
@@ -56,7 +56,7 @@
         expect(page).to have_field(:city)
         expect(page).to have_field(:state)
         expect(page).to have_field(:zip_code)
-        expect(page).to have_field(:description)
+        # expect(page).to have_field(:description)
 
```

Comment:
Should this expectation be commented out? The description and bid_reason are two separate things right?

Reply:
ah! good catch. I had commented it out mistakenly